src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.product-card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  
+  &__name {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+  
+  &:hover &__name {
+    transition: color 300ms;
+    color: #34568b;
+  }
+
+  &__code {
+    color: $secondary;
+    line-height: 14px;
+    font-size: 10px;
+    margin-top: 4px;
+  }
+
+  &__rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    &-reviews {
+      font-size: 10px;
+      text-align: center;
+      color: $main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-top: 24px;
+
+    &-text {
+      font-size: 12px;
+      color: $secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      color: $main-accent;
+      font-weight: 700;
+    }
+  }
+
+  &__buy {
+    display: block;
+    font-size: 14px;
+    font-weight: 700;
+    margin-top: 16px;
+    height: 40px;
+    line-height: 40px;
+    text-transform: uppercase;
+    text-decoration: none;
+    background-color: $blue-accent;
+    color: white;
+    text-align: center;
+    border-radius: 5px;
+    border: 1px solid $blue-accent;
+
+    &:hover {
+      transition: color 300ms, background-color 300ms;
+      background-color: white;
```

use hex values instead of reserved names
```suggestion
      background-color: #ffffff;
```